### PR TITLE
COG-216-Add User Validation with RACFID and EMAIL

### PR DESCRIPTION
**Pull request by Sushmareddy-OSI**

### CODE REVIEW

File: lib/infrastructure/http_service.rb [hunk 1]
```diff
@@ -8,8 +8,8 @@ def initialize(base_url = Rails.configuration.micro_services['perry_api_base_url
       @base_url = base_url
     end
 
-    def get(url, last_name, token)
-      http_connection.get("#{url}?lastName=#{last_name}&token=#{token}")
+    def get(url, parameters = {}, token)
```
Comment (kelfink) [inline lib/infrastructure/http_service.rb]: yes!  cleaning up the get method

**Comment by RamuKammagani-osi** [comment]
retest this please

**Comment by kelfink** [comment]
re3test this please

**Comment by kelfink** [comment]
retest this please

**Pull request merged**